packages/server/graphql/public/mutations/upsertTeamPromptResponse.ts

```diff
@@ -60,13 +59,15 @@ const upsertTeamPromptResponse: MutationResolvers['upsertTeamPromptResponse'] =
     return standardError(new Error('Invalid editor format'), {userId: viewerId})
   }
 
-  const teamPromptResponseId = await upsertTeamPromptResponseQuery({
-    meetingId,
-    userId: viewerId,
-    sortOrder: 0, //TODO: placeholder as currently it's defined as non-null. Might decide to remove the column entirely later.
-    content,
-    plaintextContent
-  })
+  const teamPromptResponseId = TeamPromptResponseId.join(
```

Comment:
-1 when we decide to move the id splitting/joining id part to DB queries level we should also return the proper id from that query so we don't have to remember to do that manually ie. let's try to keep DB queries as a single source of truth about id and do not use `TeamPromptResponseId` outside of them at all.

Reply:
That makes sense.  Moved the interop between the prefixed graphql queries and the numerical DB queries into the SQL queries themselves.

Reply:
I thought about moving into ts part but what you did looks even better, awesome work!